data.js

```diff
@@ -8,4 +8,13 @@ const peopleChoiceData = [
         favSeries: 'Mr. Robot',
         favSeriesID: '62560-mr-robot'
     },
-]
\ No newline at end of file
+    {
+        username: 'tanishqj-19',
+        favAnime: 'Jujutsu Kaisen',
+        favAnimeID: '95479',
+        favMovie: 'Fight Club',
+        favMovieID: '50-fight-club',
```

Comment:
incorrect

Reply:
It's 550 not 50

Reply:
Sorry for my mistake aditya I have updated it.